client/app/admin-dashboard/page.jsx

```diff
@@ -1,14 +1,18 @@
 import { Suspense } from "react";
 import { CircularProgress, Typography } from "@mui/material";
 import ReservationTable from "../../components/admin_dashboard/ReservationTable";
+import Map from "../../components/admin_dashboard/Map";
+import { fetchAppointments } from "@/actions/form";
 
 export default async function AdminDashboardView() {
-    return (
-        <div>
-            <Typography variant="h1">Reservations:</Typography>
-            <Suspense fallback={<CircularProgress />}>
-                <ReservationTable />
-            </Suspense>
-        </div>
-    );
+  const appointments = await fetchAppointments();
+  return (
+    <div>
+      <Typography variant="h1">Reservations:</Typography>
+      <Suspense fallback={<CircularProgress />}>
+        <ReservationTable />
```

Comment:
ReservationTable is still making a separate fetch internally. I think it'd make more sense if it also accepted an appointments prop